contrib/devtools/copyright_header.py

```diff
@@ -85,23 +85,23 @@ def get_filenames_to_examine(base_directory):
 ANY_COPYRIGHT_COMPILED = re.compile(ANY_COPYRIGHT_STYLE_OR_YEAR_STYLE)
 
 def compile_copyright_regex(copyright_style, year_style, name):
-    return re.compile('%s %s,? %s' % (copyright_style, year_style, name))
+    return re.compile('%s %s,? %s\n' % (copyright_style, year_style, name))
```

Comment:
Should this be a raw string literal now as well?

Reply:
It's not strictly required since the deprecation is about invalid escape sequences, and \n is valid. But the change in https://github.com/bitcoin/bitcoin/pull/15257/commits/45760242428fa53ba2719bbc35a68608cc4fc972 makes it necessary (because of `\*`).